File: sangi_practice/sangi_practice_03/model_answers/java/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        int cnt = 0;
        for (int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if (c == 'a' || c == 'i' || c == 'u' || c == 'e' || c == 'o') {
                cnt++;
            }
        }
        System.out.println(cnt);
    }
}

File: sangi_practice/sangi_practice_03/model_answers/cpp/main.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string s;
    cin >> s;
    int cnt = 0;
    for (char c : s) {
        if (c == 'a' || c == 'i' || c == 'u' || c == 'e' || c == 'o') {
            cnt++;
        }
    }
    cout << cnt << "\n";
    return 0;
}
